docs/CHANGELOG.asciidoc

```diff
@@ -60,6 +61,332 @@ Review important information about the {kib} 8.x releases.
+====
+      
+[float]
+[[features-8.13.0]]
+=== Features
+{kib} 8.13.0 adds the following new and notable features.
+
+Alerting::
+* The Custom Threshold rule is now out of technical preview and generally available ({kibana-pull}176514[#176514]).
+* Adds threshold to the custom threshold alert document ({kibana-pull}176043[#176043]).
+* Adds the ability to post Block Kit messages to the Slack Web API action ({kibana-pull}174303[#174303]).
+* Adds criticality fields and risk score fields to alert schema ({kibana-pull}174626[#174626]).
+* Adds fields table to rule details page alert flyout ({kibana-pull}172830[#172830]).
+APM::
+* Show Universal Profiling on Transaction view ({kibana-pull}176302[#176302]).
+* Adds a specific metrics dashboard for opentelemetry-node services ({kibana-pull}174700[#174700]).
+Dashboards::
+* Adds the creating of {esql} charts from the dashboard ({kibana-pull}171973[#171973]).
+Elastic Security::
+For the Elastic Security 8.13.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+{esql}::
+* Adds enhanced {esql} query editing experience with client side validation ({kibana-pull}170071[#170071]).
+Fleet::
+* Adds reference to `ecs@mappings` for each index template ({kibana-pull}174855[#174855]).
+* Adds support for the `subobjects` setting on the object type mapping ({kibana-pull}171826[#171826]).
+Infrastructure::
+* Adds a new Services component to host details UI ({kibana-pull}176539[#176539]).
+Integrations::
+* Adds tiles for Notion and Redis connectors ({kibana-pull}177306[#177306]).
+Lens & Visualizations::
+* Adds workspace panel dimensions by chart type ({kibana-pull}168651[#168651]).
+Machine Learning::
+* Adds the single metric viewer embeddable for dashboards ({kibana-pull}175857[#175857]).
+* Adds support for {esql} in Data visualizer ({kibana-pull}174188[#174188]).
+Management::
+* Adds a method of excluding data tiers when getting a field list ({kibana-pull}167946[#167946]).
+Reporting::
+* Adds support for 'auto' value for CSV scrolling duration ({kibana-pull}175005[#175005]).
+* Adds CSV reporting with {esql} in Discover ({kibana-pull}174511[#174511]).
+Observability::
+* Adds actions column ({kibana-pull}175872[#175872]).
+* Adds resource column with tooltip ({kibana-pull}175287[#175287]).
+* Adds support for the Timeslice Metric visualization on the SLO detail page ({kibana-pull}175281[#175281]).
+* Refactor alert table registration and change default columns ({kibana-pull}175119[#175119]).
+* Adds customization for virtual columns and add the 1st virtual column ({kibana-pull}173732[#173732]).
+* Adds a new option, Visualize this query, to the generated {esql} quires in the Elastic Assistant ({kibana-pull}174677[#174677]).
+Security::
+* Adds context information in Risk Summary ({kibana-pull}174574[#174574]).
+* Adds the Asset Criticality selector for new entity flyouts ({kibana-pull}174438[#174438]).
```

Comment:
Would these belong as part of the `{security-guide}/release-notes.html[_Elastic Security Solution Release Notes_]`?